### PR TITLE
*: update/fix Grafana boards

### PR DESCRIPTION
**Pull request by gsora**
Update grafana with the latest version off CDVN, also make prometheus track Nethermind.

Bind grafana container to 0.0.0.0 by default, since operators need to access it from the host system.

More advanced operators can still edit the grafana bind IP with the `MONITORING_IP_GRAFANA` environment variable.

### CODE REVIEW

**Review by OisinKyne — COMMENTED** [review]
Need to stop forgetting to hit submit. one thing I think we need to fix that is important

File: prometheus/prometheus.yml [hunk 1]
```diff
@@ -12,6 +12,10 @@ remote_write:
         action: keep # Keeps charon metrics and drop metrics from other containers.
 
 scrape_configs:
+  - job_name: "nethermind"
+    metrics_path: /debug/metrics/prometheus
```
[the remark this answers is not in the document]
Comment (gsora) [reply prometheus/prometheus.yml:16]: Hmm weird, this comes straight from CDVN. Will commit.

**Review by OisinKyne — APPROVED** [review]
Wasn't able to eyeball the grafana yet but if we want to merge we can. (I note this only touches the charon dash and not the full node overview dash)

**Comment by gsora** [comment]
@OisinKyne we can dump the latest versions off our internal grafana if you wish!

**Comment by KaloyanTanev** [comment]
Started the monitoring stack + Nethermind in docker compose locally. Prometheus seems to connect to Nethermind, but I get 401s, is it expected?

**Review by KaloyanTanev — APPROVED** [review]

File: prometheus/prometheus.yml [hunk 2]
```diff
@@ -12,6 +12,9 @@ remote_write:
         action: keep # Keeps charon metrics and drop metrics from other containers.
 
 scrape_configs:
+  - job_name: "nethermind"
+    static_configs:
+      - targets: ["nethermind:8008"]
```
[the remark this answers is not in the document]
Comment (OisinKyne) [reply prometheus/prometheus.yml:17]: Yeah I think this is probably a must-add for this PR, it presumably doesn't work without it?

**Review by pinebit — APPROVED** [review]

**Pull request merged**